osidb/tests/endpoints/test_affects.py

```diff
@@ -168,6 +169,7 @@ def test_affectcvss_create(self, auth_client, test_api_uri):
         )
         assert response.status_code == status.HTTP_200_OK
         assert response.data["uuid"] == cvss_uuid
+        assert response.data["comment"] == ""  # null converted to ""
```

Comment:
Could you please test this functionality in a separate test with the following scenarios?
1. string -> string
2. empty string -> empty string
3. None -> empty string

Reply:
Thanks, fixed.